gratipay/utils/__init__.py

```diff
@@ -202,12 +201,15 @@ def format_money(money):
     return format % money
 
 
-def excerpt_intro(text, length=175, append=u'…'):
-    if not text:
-        return ''
-    if len(text) > length:
-        return text[:length] + append
-    return text
+def truncate(text, target=160, append=' …'):
```

Comment:
If there's no usage with a different 'append', remove it from parameter? Otherwise may need some more test cases with this parameter?